app/views/api/v2/common_parameters/main.json.rabl

```diff
@@ -2,4 +2,4 @@ object @common_parameter
 
 extends "api/v2/common_parameters/base"
 
-attributes :created_at, :updated_at, :hidden_value
+attributes :created_at, :updated_at, :hidden_value?
```

Comment:
why the question mark? does it produce a hidden_value? key? if so, this isn't what should be here. I'd do
```ruby
node(:hidden_value){|i| i.hidden_value?}
```

not sure if that's the correct rabl syntax. point is, we need this to work like it did before.